File: 4051-RemoveZerosInDecimalRepresentation/4051-RemoveZerosInDecimalRepresentation.py
# Last updated: 31/01/2026, 07:57:39
class Solution(object):
    def removeZeros(self, n):
        x=""
        s=str(n)
        for i in range(len(s)):
            if s[i]!='0':
                x+=s[i]
        return int(x)
        

File: 4051-RemoveZerosInDecimalRepresentation/4051-RemoveZerosInDecimalRepresentation.java
// Last updated: 31/01/2026, 07:57:40
class Solution {
    public long removeZeros(long n) {
        String s="";
        while(n!=0){
            long a=n%10;
            if(a!=0){
                s=String.valueOf(a)+s;
            }
            n/=10;
        }
        return Long.parseLong(s);
    }
}